app/workers/notifications/mention_worker.rb

```diff
@@ -0,0 +1,11 @@
+module Notifications
+  class MentionWorker
+    include Sidekiq::Worker
+    sidekiq_options queue: :high_priority, retry: 10
+
```

Comment:
Not super sure about the queue here🙈 I presume sending Mentions to users should not be delayed heavily, but happy to change this :)